mpp/shmem.h.in

```diff
@@ -488,8 +492,313 @@ void *shrealloc(void *ptr, size_t size) __attribute__ ((deprecated));
 void shfree(void *ptr) __attribute__ ((deprecated));
 void start_pes(int npes) __attribute__ ((deprecated));
 
+/* C++ overloaded declarations */
+#ifdef __cplusplus
+} /* extern "C" */
+#undef complex
+
+using std::ptrdiff_t;
+using std::size_t;
+
+/* Blocking block, scalar, and block-strided put */
+static inline void shmem_put(float* dest, const float* source, size_t nelems, int pe) {
+    shmem_float_put(dest,source,nelems,pe);
```

Comment:
Sorry, this is an obnoxious nitpick.  Could you run a regex to add spaces after commas?